scripts/test.backend.sh

```diff
@@ -15,6 +17,15 @@ else
   cargo build --locked --target wasm32-unknown-unknown --release -p backend
 fi
 
+if [ -f "./$BITCON_CANISTER_WASM" ]; then
+  echo "Use existing $BITCON_CANISTER_WASM canister."
+else
+  echo "Downloading bitcoin_canister canister."
+  curl -sSL "https://github.com/dfinity/bitcoin-canister/releases/download/release%2F$BITCOIN_CANISTER_RELEASE/ic-btc-canister.wasm.gz" -o $BITCON_CANISTER_WASM
+fi
+# Setting the environment variable that will be used in the test to load that particular file relative to the cargo workspace.
+export BITCOIN_CANISTER_WASM_FILE="../../$BITCON_CANISTER_WASM"
```

Comment:
Note that if you put code such as the above in a script, you can do things such as have `dfx deploy` "just work".  See e.g. the cycles depositor script here: https://github.com/dfinity/chain-fusion-signer/blob/main/scripts/build.cycles_depositor.sh  Running `dfx build cycles_depositor` runs that script.

Reply:
I am not sure I want the bitcoin canister in dfx.json. The bitcoin canister should not be installed in the local replica. To have bitcoin in the local replica we need to use a flag from the command line. Then dfx installs the bitcoin canister.